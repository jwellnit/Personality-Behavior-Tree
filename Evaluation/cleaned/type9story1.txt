Tom collects the medicinal herbs from the cottage.
All Characters wait.
All Characters wait.
Tom goes to the town.
All Characters wait.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
The alchemist comes back to the shop.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
All Characters wait.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
Tom steals the recipe WITHOUT anyone noticing.
The alchemist comes back to the shop.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
Tom goes to the town.
All Characters wait.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
The alchemist comes back to the shop.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
The alchemist comes back to the shop.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
All Characters wait.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
The alchemist comes back to the shop.
All Characters wait.
All Characters wait.
Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
Tom steals the recipe from the alchemist.
The alchemist comes back to the shop.
Tom steals the recipe from the alchemist.
The alchemist calls for the guard.
Tom steals the recipe from the alchemist.
The alchemist calls for the guard.
The guard arrests Tom.