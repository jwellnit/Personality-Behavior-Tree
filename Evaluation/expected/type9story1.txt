Tom goes to the town.
Tom knocks on the door and the alchemist lets him into the shop.
Tom tells the alchemist about the herbs in the cottage.
The alchemist leaves the shop to collect the herbs.
Tom steals the recipe WITHOUT anyone noticing.
The alchemist collects the herbs.
The alchemist comes back to the shop.
Tom steals the herbs from the alchemist.
Tom makes the medicine using the herbs.
The alchemist calls for the guard.
The guard arrests Tom.
